src/styles/blocks/card.css

```diff
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--background-color);
+  border: solid #f3f3f3 1px;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__code {
+  align-self: flex-start;
+  margin-bottom: 16px;
+
+  line-height: 14px;
+  color: var(--text-second-color);
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+
+  font-style: 12px;
+  line-height: 18px;
+}
+
+.price-name {
+  color: var(--text-second-color);
+}
+
+.price-value {
```

Comment:
These elements shouldn't be separate BEM block, check correct class names for making them card elements